db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+FEATURE_FLAGS = %i[make_session_in_progress_button new_consents].freeze
+
+FEATURE_FLAGS.each { |flag| Flipper.add(flag) unless Flipper.exist?(flag) }
```

Comment:
Ah, missed this step. Is there a specific reason, though, not to just them auto-create?

Reply:
Yeah, you get a warning to STDOUT every time you visit a page with a new feature flag. It makes the test output spammy